packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -10,6 +10,7 @@ import LOCALE, { DEFAULT_TICK_FORMAT } from '../locale';
 import { prepareHAxis } from './datetimeBuckets';
 import { shouldTruncateLabel, truncateLabel } from './label';
 import * as localeFormat from './localeFormat';
+// import { cosh } from 'core-js/core/number';
```

Comment:
Is this still needed?

Reply:
Removed here: c70a5e131bc72a698715c2ff2ac0023705d22a46